API.md

```diff
@@ -1516,3 +1538,57 @@ this._viewer.getZoom().then((zoom) => {
   console.log('Zoom scale of the current document is:', zoom);
 });
 ```
+
+### Coordinate
+
+#### convertPoints
+Converts points between multiple coordinate systems of the viewer.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+points | array | list of points, each in the format `{x: number, y: number}`. If either the source or destination coordinate system is page, you could optionally have a `pageNumber: number` in the object. Without specifying, the page system is referring to the current page
+from | string | One of the [Config.Conversion](./src/Config/Config.js) constants, representing the source coordinate system
+to | string | One of the [Config.Conversion](./src/Config/Config.js) constants, representing the destination coordinate system
+
+Returns a Promise.
+
+Promise Parameters:
+
+Name | Type | Description
+--- | --- | ---
+convertedPoints | array | list of converted points in the specified destination system, each in the format `{x: number, y: number}`. It would be empty if conversion is unsuccessful
+
+```js
+// convert (50, 50) in current page and (100, 100) on page 1 in page system to canvas system
+this._viewer.convertPoints([{x: 50, y: 50}, {x: 100, y:100, pageNumber: 1}], Config.Conversion.Page, Config.Conversion.Canvas).then((convertedPoints) => {
```

Comment:
don't use canvas as example, it really shouldn't be used, use screen and page point as sample only, i'd recommend we remove it entirely to not confuse customers